ui/desktop/README.md

```diff
@@ -103,11 +105,12 @@ These environment variables may be used to customized the build.
 | `BYPASS_APP_UPDATER` | Disable app updater feature. For development use only. |
 | `DISABLE_WINDOW_CHROME` | Disable window chrome. For internal use only. |
 | `ENABLE_MIRAGE` | Enable (`true`) or disable (`false`) mirage. Default value is `true`. |
+| `DARK_MODE` | Enable (`true`) to set the test browser appearance to dark for running tests in dark mode. |
```

Comment:
What's the difference between `DARK_MODE` and `COLOR_THEME`?

Reply:
Apologies. We originally used `DARK_MODE` as the flag name but later changed it to `COLOR_THEME` and I didn't catch it in the README. Updating now.